commit c4f71901d53b6d8a4703389459d9f99fbd80ffd2
Merge: fd49e8ee70b3 9a8aae605b80
Author: Paolo Bonzini <pbonzini@redhat.com>
Date:   Fri Apr 23 07:41:17 2021 -0400

    Merge tag 'kvmarm-5.13' of git://git.kernel.org/pub/scm/linux/kernel/git/kvmarm/kvmarm into HEAD
    
    KVM/arm64 updates for Linux 5.13
    
    New features:
    
    - Stage-2 isolation for the host kernel when running in protected mode
    - Guest SVE support when running in nVHE mode
    - Force W^X hypervisor mappings in nVHE mode
    - ITS save/restore for guests using direct injection with GICv4.1
    - nVHE panics now produce readable backtraces
    - Guest support for PTP using the ptp_kvm driver
    - Performance improvements in the S2 fault handler
    - Alexandru is now a reviewer (not really a new feature...)
    
    Fixes:
    - Proper emulation of the GICR_TYPER register
    - Handle the complete set of relocation in the nVHE EL2 object
    - Get rid of the oprofile dependency in the PMU code (and of the
      oprofile body parts at the same time)
    - Debug and SPE fixes
    - Fix vcpu reset

diff --cc arch/arm64/kvm/arm.c
index 0d92a4e5fe80,4808aca8c87c..1cb39c0803a4
--- a/arch/arm64/kvm/arm.c
+++ b/arch/arm64/kvm/arm.c
@@@ -206,10 -206,9 +206,11 @@@ int kvm_vm_ioctl_check_extension(struc
  	case KVM_CAP_ARM_INJECT_EXT_DABT:
  	case KVM_CAP_SET_GUEST_DEBUG:
  	case KVM_CAP_VCPU_ATTRIBUTES:
+ 	case KVM_CAP_PTP_KVM:
  		r = 1;
  		break;
 +	case KVM_CAP_SET_GUEST_DEBUG2:
 +		return KVM_GUESTDBG_VALID_MASK;
  	case KVM_CAP_ARM_SET_DEVICE_ADDR:
  		r = 1;
  		break;